src/wallet/spend.cpp

```diff
@@ -403,6 +403,20 @@ std::optional<SelectionResult> AttemptSelection(const CWallet& wallet, const CAm
         results.push_back(*srd_result);
     }
 
+    // If we didn't find any solutions, and are subtracting the fee from the outputs, we may need to include negative effective value coins.
+    // So try BnB and SRD again with all_groups.
+    // It is safe to do this because a solution including negative EVs will have a greater waste.
```

Comment:
What about just throwing these two selections into the mix unconditionally? Would that be correct? Efficiency aside.